engine.js

```diff
@@ -203,6 +177,13 @@ HMDAEngine.prototype.getFileSpec = function(year) {
     return hmdaRuleSpec.getFileSpec(year);
 };
 
+/**
+ * Get the progress object used for task completion events displayed on the progress bar
+ * @return {object} Progress object containing an eventemitter for progress notification
+ */
+HMDAEngine.prototype.getFileProgress = function() {
+    return hmdajson.getProgress();
+};
```

Comment:
Why would you create a convenience for functions that are already exposed since `ruleProgress` is a mixin to the engine and therefore are part of the engines prototype already?